brave/brave-core
# Import number of accounts

Resolves https://github.com/brave/brave-browser/issues/18380

1. Remove `KeyringController` from `BraveWalletServiceDelegateImpl` to simplify what delegate should do, `BraveWalletService` should access `KeyringController` directly through the pointer factory passed in.
2. Read `data.numberOfAccounts` from CW or MM and derived the same number of accounts after auto import

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Setup MetaMask or Crypto Wallets
2. Add 4 accounts so it has 5 in total
3. Enable brave://flags/#native-brave-wallet
4. Import from MetaMask or Crypto Wallets
5. We should have same number of accounts
